packages/aws-cdk-lib/core/lib/prop-injectors-helpers.ts

```diff
@@ -19,6 +20,8 @@ export function applyInjectors(uniqueId: string, originalProps: any, context: In
     // no injector found
     return originalProps;
   }
+  const stack = Stack.of(context.scope);
+  stack.addMetadata('@amzn/aws-cdk-lib-blueprint', 'Y');
```

Comment:
- must consider opt-out
- naming
- we have different metadata system already, can we use that?